### PR TITLE
Add ADR to record government and political tagging decision

### PR DESCRIPTION
**Pull request by kevindew**
This is to supersede the previous ADR that explained our decision to not
implement this tagging.

[Rendered version](https://github.com/alphagov/content-publisher/blob/government-rfc/docs/adr/00XX-political-and-government-tagging.md)

### CODE REVIEW

File: docs/adr/00XX-political-and-government-tagging.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# XX. Political and Government Tagging
```
Comment (ChrisBAshton) [inline docs/adr/00XX-political-and-government-tagging.md]: Any reason why we haven't committed to a number yet?

Comment (kevindew) [reply docs/adr/00XX-political-and-government-tagging.md]: Yes - in the past these have sometimes dragged on for a while and the numbers have become conflicting.

Comment (ChrisBAshton) [reply docs/adr/00XX-political-and-government-tagging.md]: If I approve, will you edit the number before merging or does it happen in a separate PR?

Comment (kevindew) [reply docs/adr/00XX-political-and-government-tagging.md]: Yup before merging.

File: docs/adr/0006-tagging-governments.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # 6. Tagging Content to Governments
 
 Date: 2018-12-12
+Superseded: 2019-12-19 by [ADR XX](00XX-political-and-government-tagging.md)
```
Comment (ChrisBAshton) [inline docs/adr/0006-tagging-governments.md]: Whoops @kevindew ^

**Pull request merged**